tests/kani/Intrinsics/Atomic/Unstable/AtomicMin/main.rs

```diff
@@ -1,12 +1,13 @@
 // Copyright Kani Contributors
 // SPDX-License-Identifier: Apache-2.0 OR MIT
 
-// Check that `atomic_min` and other variants (unstable version) return the
+// Check that `atomic_min_seqcst` and other variants (unstable version) return the
 // expected result.
 
 #![feature(core_intrinsics)]
 use std::intrinsics::{
-    atomic_min, atomic_min_acq, atomic_min_acqrel, atomic_min_rel, atomic_min_relaxed,
+    atomic_min_acqrel, atomic_min_acquire, atomic_min_relaxed, atomic_min_release,
```

Comment:
Looks like the new namespace may include more options for atomic intrinsics: something for @adpaco-aws to look at?

Reply:
I created https://github.com/model-checking/kani/issues/1344 for that.